### ISSUE TITLE
Migrating Angular applications from Spa Extensions

### ISSUE THREAD

**Issue by Rick-Anderson** [dotnet/AspNetCore.Docs#27772]
see [Migrating Angular applications from Spa Extensions](https://github.com/dotnet/aspnetcore/issues/44600)

see #17902

related #27824

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 76f39eec-9dd1-6167-56ec-47b26e0b5063
* Version Independent ID: f9f2b0f3-01eb-cf58-8767-194fe0fc0e44
* Content: [Migrate from ASP.NET Core 2.2 to 3.0](https://learn.microsoft.com/en-us/aspnet/core/migration/22-to-30?view=aspnetcore-7.0&tabs=visual-studio#add-package-references-for-removed-assemblies)
* Content Source: [aspnetcore/migration/22-to-30.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/migration/22-to-30.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-migration**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
@rachelgshaffer @ajaybhargavb @peterblazejewicz @sgbj @speige would any of you be able to review and answers questions as I migrate [Migrating Angular applications from Spa Extensions](https://github.com/dotnet/aspnetcore/issues/44600) to the docs?

**Comment by peterblazejewicz** [comment]
@Rick-Anderson I'm using now microfronteds (Angular + .Net Core stack with custom middlewares), but I can do the proofreading/testing
TL;DR I can help

**Comment by Rick-Anderson** [comment]
> @Rick-Anderson I'm using now microfronteds (Angular + .Net Core stack with custom middlewares), but I can do the proofreading/testing TL;DR I can help

That sounds interesting. Do you have a  blog link or other doc that explains that approach?

**Comment by peterblazejewicz** [comment]
@Rick-Anderson 
We are using approach promoted by Angular Architects, e.g.:
https://github.com/angular-architects/module-federation-plugin
https://www.angulararchitects.io/en/aktuelles/interview-micro-frontends-and-module-federation-at-cube-bikes/
https://www.angulararchitects.io/en/aktuelles/the-microfrontend-revolution-part-2-module-federation-with-angular/
we have microfrontends + .Net Core API microservices plus everything else backed with orchestrated Azure services (one client module per app domain slice plus a shell app).
When I read something describing exactly same architecture, I'll make sure I referenced you
